frontend/components/modal/ModalSharePage.vue

```diff
@@ -90,6 +90,32 @@
             :use-native-behavior="useNativeBehavior"
             :native-behavior-options="nativeBehaviorOptions"
           />
+          <BtnShareIcon
+            type="redirect"
+            :iconName="IconMap.INSTAGRAM"
+            :text="$t('components._global.instagram')"
+            iconSize="1.5em"
+            :window-features="windowFeatures"
+            :share-options="shareOptions"
+            :use-native-behavior="useNativeBehavior"
+            :native-behavior-options="nativeBehaviorOptions"
+            :urlLink="getCurrentUrl()"
+            :name="getCurrentName()"
+            redirect-link="https://signal.me/#p"
```

Comment:
This is copied from the Signal entry and should be swapped over to something for Instagram, right?